src/formatDate.test.js

```diff
@@ -53,6 +53,19 @@ describe('formatDate', () => {
         .toBe('21-12-2012');
     });
 
+    it(`for non standard parts order '12-2012-21'
```

Comment:
when you write tests it is better to have more clear description
But good initiative to learn writing tests 👍